api/jobs/api.py

```diff
@@ -217,9 +217,14 @@ def list(self, request):
 
         # SOC codes in transitions data that are close to an exact match to the keyword - tiered matching, since O*NET
         # does not include older SOC codes that exist in the transitions data
+        for soc_entry in available_socs:
```

Comment:
Can you change the log on line 206 to log.debug?

Reply:
Changed!